helm-chart/kuberay-operator/templates/ray_raycluster_role.yaml

```diff
@@ -0,0 +1,38 @@
+# permissions for end users to view rayjobs.
+{{- if and .Values.rbacEnable (not .Values.singleNamespaceInstall) }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: raycluster-viewer-role
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
```

Comment:
according to https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go#L290C1-L291C1, and https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go#L280 view clusterrole will aggregated to edit clusterrole, and edit clusterrole will aggregated to admin clusterrole, so only rbac.authorization.k8s.io/aggregate-to-view is enough.